client/my-sites/checkout/composite-checkout/wpcom/components/wp-checkout-order-summary.js

```diff
@@ -145,6 +146,42 @@ function CheckoutSummaryFeaturesListDomainItem( { domain } ) {
 	);
 }
 
+function CheckoutSummaryPlanFeatures() {
+	const translate = useTranslate();
+	const hasDomainsInCart = useHasDomainsInCart();
+	const planInCart = usePlanInCart();
+	const planFeatures = getPlanFeatures( planInCart, translate, hasDomainsInCart );
+
+	return (
+		<>
+			{ planFeatures &&
+				planFeatures.map( ( feature ) => {
+					return (
+						<CheckoutSummaryFeaturesListItem>
+							<WPCheckoutCheckIcon />
+							{ feature }
+						</CheckoutSummaryFeaturesListItem>
+					);
+				} ) }
+		</>
+	);
+}
+
+function getPlanFeatures( plan, translate, hasDomainsInCart ) {
+	if (
+		'business-bundle' === plan.wpcom_meta?.product_slug ||
+		'business-bundle-2y' === plan.wpcom_meta?.product_slug
+	) {
+		return [
+			! hasDomainsInCart && translate( 'Free domain for one year' ),
+			translate( 'Install plugins and themes' ),
+			translate( 'Drive traffic to your site with our advanced SEO tools' ),
+			translate( 'Track your stats with Google Analytics' ),
+		];
+	}
+	return false;
```

Comment:
Why `false`? I suggest `null` as a failure value, as it's more semantic to have a type like `?string[]` than `string[] | false`. Even better, just return an empty array. Then you can remove the check for this return value and just use `map` on it since `map` on an empty array is a noop.